### ISSUE TITLE
[3.0]: Upgrader fails on "adding support for recurring events" step

### ISSUE THREAD

**Issue by Oldiesmann** [SimpleMachines/SMF#8256]
### Basic Information

Trying to run upgrade.php on SMF 3.0 results in an SQL syntax error

### Steps to reproduce

1. Copy appropriate files to forum root
2. Run upgrade.php either from browser or CLI

### Expected result

Upgrade finishes successfully

### Actual result

```
You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '(0) DEFAULT 'NULL'' at line 2
```

### Version/Git revision

c909204

### Database Engine

All

### Database Version

10.11.8

### PHP Version

8.3.8

### Additional Information

Running this on CLI gives you slightly more details - it occurs immediately after it says "Adding support for recurring events...".
Also note that this is occurring on a forum that already has the recurring events stuff.

**Comment by Oldiesmann** [comment]
Further info: This seems to only occur on MariaDB/MySQL - I tried running upgrade.php on my Postgres test board and it ran fine with no errors.

**Comment by Oldiesmann** [comment]
Did some further digging and think I got to the bottom of this. Checking the database for the forum in question, I still have the old calendar_holidays table even though everything has been converted to the new system. I believe this is because I manually ran some of the queries to create the new events table (or update it anyway) when the code for that was initially added due to other bugs, and I forgot to drop that table once everything got converted. On that note, I'll close this issue.
